README.md

```
@@ -33,12 +34,12 @@ Each addon directory contains its own README with specific installation and usag
 To onboard a new addon-contrib project:
 
 1. If not already discussed with maintainers, open an issue to propose your idea.
-1. Once acknowledged, create a folder named after your project and add your code/docs.
-1. Add an `OWNERS` file listing the new project's maintainers.
-1. **Register your project** in `.github/repositories.json` (see [Repository Registration](#repository-registration) below).
-1. **Ensure CI/CD compliance** by implementing required structure and targets (see [GitHub Actions Requirements](#github-actions-requirements) below).
-1. Create a PR with a brief project overview and confirm all requirements are met.
-1. An OCM maintainer will review and merge the PR.
+2. Once acknowledged, create a folder named after your project and add your code/docs.
+3. Add an `OWNERS` file listing the new project's maintainers.
+4. **Register your project** in `.github/repositories.json` (see [Repository Registration](#repository-registration) below).
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)